src/test/java/us/kbase/test/auth2/lib/TemporarySessionDataTest.java

```diff
@@ -52,31 +53,33 @@ public void constructLoginStart() throws Exception {
 		assertThat("incorrect expires", ti.getExpires(), is(inst(20000)));
 		assertThat("incorrect state", ti.getOAuth2State(), is(opt("stategoeshere")));
 		assertThat("incorrect pkce", ti.getPKCECodeVerifier(), is(opt("pkcegoeshere")));
-		assertThat("incorrect user", ti.getUser(), is(Optional.empty()));
-		assertThat("incorrect idents", ti.getIdentities(), is(Optional.empty()));
-		assertThat("incorrect error", ti.getError(), is(Optional.empty()));
-		assertThat("incorrect error type", ti.getErrorType(), is(Optional.empty()));
+		assertThat("incorrect user", ti.getUser(), is(opt()));
+		assertThat("incorrect idents", ti.getIdentities(), is(opt()));
+		assertThat("incorrect error", ti.getError(), is(opt()));
+		assertThat("incorrect error type", ti.getErrorType(), is(opt()));
 		assertThat("incorrect has error", ti.hasError(), is(false));
+		assertThat("incorrect mfa", ti.getMFA(), is(opt()));
 	}
 	
 	@Test
 	public void constructLoginIdents() throws Exception {
 		final UUID id = UUID.randomUUID();
 		final Instant now = Instant.now();
 		final TemporarySessionData ti = TemporarySessionData.create(
-				id, now, now.plusMillis(100000)).login(set(REMOTE1, REMOTE2));
+				id, now, now.plusMillis(100000)).login(set(REMOTE1, REMOTE2), MFAStatus.USED);
 		
 		assertThat("incorrect op", ti.getOperation(), is(Operation.LOGINIDENTS));
 		assertThat("incorrect id", ti.getId(), is(id));
 		assertThat("incorrect created", ti.getCreated(), is(now));
 		assertThat("incorrect expires", ti.getExpires(), is(now.plusMillis(100000)));
 		assertThat("incorrect state", ti.getOAuth2State(), is(ES));
 		assertThat("incorrect pkce", ti.getPKCECodeVerifier(), is(ES));
-		assertThat("incorrect user", ti.getUser(), is(Optional.empty()));
+		assertThat("incorrect user", ti.getUser(), is(opt()));
```

Comment:
I would not immediately know `opt()` vs `Optional.empty` meant an empty optional, but I also am not steeped in Java norms.